viaMover/web-app
# Governance: Add basic UI components, views & layouts

What was done
* Added a set of common nav components (emoji, item links)
* Added new views, components to the governance
* Updated i18n map with new localized strings
* Updated router's schema to add another level deep
* Prepared pages to be developed by the team

How to test
* set `isGovernanceEnabled` to `true`
* check the pages

TODO/Notes
* New assets for left-rail: TBA
* Route guards for each and every subroute using dynamic item match and/or fetch / fallbacks: TBA
* Snapshot lib: TBA
